### DISCUSSION TITLE
Wyze Cam V4 Is out! Anyone order one? Update: Ordered for @gtxaspec

### DISCUSSION THREAD

**Discussion by evanheckert** [gtxaspec/wz_mini_hacks#750] [General]
Here's hoping wz_mini_hacks can work on one. The spec bump looks great, as long as local-only is an option!

**Comment by digitydogs** [comment]
I have a few ordered and on the way, Waiting to hear if they are supported or not.  Really hoping they are though!

**Comment by gtxaspec** [comment]
the key is opening one up and analyzing the hardware to see if it's possible.  I know its Ingenic based, so its a possibility, unless secure boot is enabled.

If anyone wants to send one my way, feel free =D https://www.amazon.com/hz/wishlist/ls/C4VJ2DZ3F5A0

**Reply by evanheckert** [reply]
Sure thing, just ordered for you. Fastest shipping it let me pick was April 8th.

**Reply by evanheckert** [reply]
Out for delivery today!

**Reply by gtxaspec** [reply]
Wow thanks a lot!  I'll have photos and info up ASAP!

**Reply by gtxaspec** [reply]
just got it!

**Reply by digitydogs** [reply]
Can't wait to hear what you discover! Fingers Crossed they didn't enable secure boot.  

If they did, what do we know about their implementation?  

Thinking about how we can crack the secure u-boot on many android systems and replace it with our own signed/unsigned versions, maybe something similar could be worked out for wyze's implementations.

**Reply by claudobahn** [reply]
Notes from gtxaspec are here: See https://github.com/gtxaspec/wz_mini_hacks/issues/760 - secure boot is enabled

**Reply by evanheckert** [reply]
darth-vader-noooo.gif

**Reply by JamAndCheese** [reply]
darth-vader-noooooooooooooooooo.gif

I just got one at a Sales Bin Store, for only $1.  Seems to work and I have added it to my Wyze acct.
I was going to ask what you guys need to make a determination.

**Comment by digitydogs** [comment]
Death to secure boot! 

another possible helper git 
https://github.com/OpenNoahEdu/ingenic-u-boot

**Comment by evanheckert** [comment]
> Wow thanks a lot! I'll have photos and info up ASAP!

I saw the note on secure boot. Does that just mean we're dead in the water? @gtxaspec

**Reply by gtxaspec** [reply]
maybe, but hopefully not for long.  the wheels grind slowly =D

**Comment by gtxaspec** [comment]
root has been achieved.  secureboot is still there, but its a step.

**Reply by endertable** [reply]
Wow!!  Fantastic!  Is this through thingino or stock Wyze FW?

**Reply by ghost** [reply]
That's extremely exciting! Looking forward to more :)

**Reply by ghost** [reply]
> root has been achieved. secureboot is still there, but its a step.

Just wanted to check in and see you've had any more luck!

**Reply by JamAndCheese** [reply]
Do you need any sort of help? I don't have a lot of free time, but I have one of them buggers... and I am willing to poke around it.
